Plugin/Catalog/Block/Toolbar.php

```diff
@@ -191,10 +204,20 @@ private function getCmpResult(Store $store, FulltextCollection $collection)
         $nostoCustomer = $this->cookieManager->getCookie(NostoCustomer::COOKIE_NAME);
         $limit = $collection->getSize();
         $personalizationResult = null;
+
+        // Get filters used
+        $this->nostoFilterMapper->init($store);
+        $selectedFilters = $this->state->getActiveFilters();
+        foreach($selectedFilters as $filter){
```

Comment:
Spacing. Run the PHPCS fixer.
```suggestion
        foreach ($selectedFilters as $filter) {
```